shared/components/HTML/index.js

```diff
@@ -8,13 +8,13 @@ import PropTypes from 'prop-types';
  * The is the HTML shell for our React Application.
  */
 function HTML(props) {
-  const { htmlAttributes, headerElements, bodyElements, appBodyString } = props;
+  const { htmlAttributes, headerElements, bodyElements, children } = props;
 
   return (
     <html {...htmlAttributes}>
       <head>{headerElements}</head>
       <body>
-        <div id="app" dangerouslySetInnerHTML={{ __html: appBodyString }} />
+        {children}
```

Comment:
this should remain with this
`<div id="app">{children}<div>`
to have consistent element between server and client render